### DISCUSSION TITLE
Difference between WasmJs and Js?

### DISCUSSION THREAD

**Discussion by ValleyDragon888** [littlektframework/littlekt#238] [Q&A] [answered]
What is the difference between the WasmJs platform and the Js platform?

**Comment by LeHaine** [comment]
Hello! `WasmJs` targets WebAssembly while `Js` targets plain old JavaScript.

**Reply by ValleyDragon888** [reply]
Are they both server-side, or is one of them client-side? Which is faster?

**Reply by LeHaine** [reply, marked as answer]
For our case, they are both client-side. Wasm is quite a lot faster than JS as well.
